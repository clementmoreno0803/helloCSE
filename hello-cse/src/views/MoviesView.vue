<template>
  <div class="home">
    <top-movie-carousel></top-movie-carousel>
    <all-movies></all-movies>
  </div>
</template>

<script setup lang="ts">
import topMovieCarousel from "@/components/movies/top-movie-carousel.vue";
import allMovies from "@/components/movies/all-movies.vue";
</script>

<style lang="scss" scoped>
.home {
  width: 95vw;
  @include marginCenter(0);
}
</style>
