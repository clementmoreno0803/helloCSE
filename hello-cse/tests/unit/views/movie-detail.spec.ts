import { describe, it, expect } from 'vitest';
import { shallowMount } from "@vue/test-utils";
import MoviesView from "@/views/MoviesView.vue";
import MovieDetail from "@/views/MovieDetail.vue";

const generateWrapper = () => shallowMount(MovieDetail, {});

describe('MoviesView', () => {
  it('should render correctly', () => {
    const wrapper = generateWrapper();
    expect(wrapper.element).toMatchSnapshot();
  });
});
