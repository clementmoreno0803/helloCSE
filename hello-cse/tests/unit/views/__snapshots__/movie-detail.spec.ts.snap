// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MoviesView > should render correctly 1`] = `
<div>
  <movies-details-stub />
</div>
`;
